### ISSUE TITLE
Contestant/LoginViewModel

### ISSUE THREAD

**Issue by Yorchi** [legacybass/Jeopardy#3]
Hi, thanks for this awesome game,

this issue if for i getting this error, 

```
Uncaught Error: Script error for: viewmodels/Contestant/LoginViewModel
```

you could help me fix this?

Thanks in advance!

**Comment by legacybass** [comment]
It looks like a missed commit, but I'll look into it. Unfortunately this is a side project I'm doing to help my students review for tests, and so doesn't have a super high priority right now. It is currently in a refactor state, so there are a number of bugs. If you'd like a working but older version, there is a 1.0 release branch that works. It's uglier, but it does do what it's supposed to.

I have a number of commits to push up to github, though, so hopefully that will fix it. Thank you for letting me know. I'll try to have it solved as soon as I can.

**Comment by nealterrell** [comment]
Any chance you add that commit, @legacybass? I'd like to use this project for my classroom, and it seems like a shame to leave it in a so-close-to-usable state.

Edit: I see there's a Dev branch with a fix. Thanks!

**Comment by legacybass** [comment]
Yeah, that's where my current work is. I haven't touched this in a while, though, because of other projects that took priority. I'm teaching a class on JavaScript this Spring semester, though, so I'm hoping to get back on this.

A word of caution, though, the Dev branch isn't fully working either. You can load up the page, but no one will be able to connect. You'll have to resort to raising hands or other buzzers for it to work properly.

**Comment by nealterrell** [comment]
Thanks for the heads up. I ran into that issue last night and figured you were still working on things. From my end, a Contestant can join the game just fine, and can Buzz in, but the Answer page view's "Did XX get the question right?" toast buttons don't do anything. I'll be using it for finals review today using hands & manual points as suggested. 

This is a fun project, I hope you're able to finish it :).
